metric_tank/http.go

```diff
@@ -137,7 +135,7 @@ func Get(w http.ResponseWriter, req *http.Request, metaCache *MetaCache, aggSett
 		http.Error(w, err.Error(), http.StatusInternalServerError)
 		return
 	}
-	reqs, err = alignRequests(reqs, metrics.MinSpan(), aggSettings)
```

Comment:
this was already removed in https://github.com/raintank/raintank-metric/commit/26f84eafd5c454af856356f6cc3cad88f7a22cb6